x-pack/plugins/security_solution/cypress/integration/alerts.spec.ts

```diff
@@ -34,16 +35,13 @@ import { DETECTIONS_URL } from '../urls/navigation';
 describe('Alerts', () => {
   context('Closing alerts', () => {
     beforeEach(() => {
+      cleanKibana();
+      removeSignalsIndex();
```

Comment:
This is a bit of a tangent, but I just had a couple thoughts:

* Keeping test setup/teardown symmetrical is a PITA
  * reorienting this to do teardown, then setup _all within setup_ would reduce this complexity
    * This would address/sidestep the issue of test failures, which can currently cause teardown not to execute
    * We could lessen developer burden by defining this as a general function(s)

```ts
// in test file
describe('my test', () => {
  detectionEngineHooks();
  
  it('does a thing', () => {});
});

// in helper file
const detectionEngineHooks = () => {
  beforeEach(() => {
    cleanKibana(); // deletes `.kibana*`, loads `empty_kibana`
    cleanSignals(); // deletes `.siem-signals*`, hits detections endpoint to create index
    cleanLoad('alerts'); // see below
  });
};

const cleanLoad = (archive) => {
    unload(archive); // might need to try/catch this?
    load(archive);
}
```